### ISSUE TITLE
Proposed change to membership criteria: fulfil student obligations first

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/london-membership#76]
Our criteria for membership may be too loose. In particular, we could require that in future students complete the 16-week programme and fulfil their commitment to mentor on at least one week on the subsequent cohort before becoming full (blocking) members.

**Comment by NataliaLKB** [comment]
> Our criteria for membership may be too loose

Why do you think so?

**Comment by claireinez** [comment]
As the current rules were designed at the AGM back in January, I think this would be an appropriate time to review how they have worked out, but I would be interested to hear why you would be in favour of extending the timeline. This seems to contradict what @bradreeder was suggesting back in July in #52 .

**Comment by sofer** [comment]
> Our criteria for membership may be too loose

Yeah. let me try to make my thinking a bit clearer. I have been thinking about this off and on and somehow it doesn't quite make sense in my mind for somebody to be a full member before they have actually completed the course. It's not that it is actually a problem right now, but I could imagine it feeling inappropriate (and irritating) if somebody for whatever reason became a member while participating on the course, but then failed to complete their commitment as [referenced in the charter](https://github.com/foundersandcoders/charter) to *"take the lead on facilitating the programme for the next student cohort."*

Most cooperatives have some sort of probationary period. All students are *de jure* associate members as soon as they start on the programme and they are often in the majority at meetings. So, not being a full member would not exclude students from participating fully in meetings.

Having a special meeting (perhaps followed by dinner) at the half way stage of each course (the next one would be in a couple of weeks) where new members (in this case from FAC11) could be inducted into the members' register would be a nice thing to do, too.

**Comment by bradreeder** [comment]
Only posting as I noticed I was tagged. @sofer do you think adding something to the membership criteria along the lines of 'will be revoked if you don't stick to the student agreement' would address this?

**Comment by sofer** [comment]
@bradreeder this just seems punitive. It would be nice if becoming a member could be something to celebrate--at the right time. Is there really an urgency to get people signed up quickly?

**Comment by NataliaLKB** [comment]
> Is there really an urgency to get people signed up quickly

Urgency isn't the right word. From what I remember from the AGM the feeling (which I agree with) is that current students have insight and value which they can bring in to FAC. 

Given this was decided at an AGM, I also think we would need to come up with a proposal process similar to that one for the next AGM. This is a decision I think the entire community should weigh in on.

**Comment by claireinez** [comment]
> This is a decision I think the entire community should weigh in on.

Agreed. There were a lot of different opinions on this back in January so I think it would need to be discussed at the AGM, rather than less well-attended business meetings.

**Comment by sofer** [comment]
To emphasise, there isn't any urgency with this. I just wanted to get the mood of the committee before proceeding. The AGM seems the right place to bring this up, but I wouldn't want to do so without getting a broadly positive or at least neutral response here.

**Comment by sofer** [comment]
@NataliaLKB @claireinez 

Agreed at last night's meeting:
- Every student is immediately eligible for associate membership
- All they have to do is make a PR (under the "Associate" sub-head) and attend a meeting
- Their membership will be merged into master at that meeting
- They can vote at meetings, but not block
- Associate members can become full members as soon as they have graduated and fulfilled their obligations to the next cohort (usually within two months of graduating) by making another PR (remove their name under "Associate" and add it under "Full") and attending another meeting

To do: confirm that this is what we agreed. Make a PR ("not ready to merge"). Raise an issue in london-programme. Mark it with the "AGM" label.

I think that was it.

**Comment by claireinez** [comment]
This PR needs to be ready for the AGM and we might want to have a refresher conversation before then. What date will that be?

**Comment by sofer** [comment]
@claireinez I am aiming for an AGM in May.

**Comment by claireinez** [comment]
Ah cool, I had assumed it would be imminent!
